pkg/clusteragent/languagedetection/patcher.go

```diff
@@ -54,12 +57,19 @@ func newLanguagePatcher(ctx context.Context, store workloadmeta.Component, logge
 	ctx, cancel := context.WithCancel(ctx)
 
 	k8sClient := apiCl.DynamicCl
+
 	return &languagePatcher{
 		ctx:       ctx,
 		cancel:    cancel,
 		k8sClient: k8sClient,
 		store:     store,
 		logger:    logger,
+		queue: workqueue.NewRateLimitingQueue(
```

Comment:
If initialized with some config, we can have some more telemetry. Example: https://github.com/DataDog/datadog-agent/blob/49cb38b22e34d56fe91d3b130326fada56727245/pkg/sbom/scanner/scanner.go#L56-L65